AnimalsTest/Lion.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace AnimalsTest{
+    
+    public class Lion : IBigCats
+    {
+
+        private string? _name;
+        public string? Name
```

Comment:
By implementation, name is always set. 
```suggestion
        public string Name
```